pages/api/v1/fantasy/index.ts

```diff
@@ -0,0 +1,87 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import SQL from 'sql-template-strings';
+import Cors from 'cors';
+import { query } from '../../../../lib/db';
+import use from '../../../../lib/middleware';
+
+const cors = Cors({
+  methods: ['GET', 'HEAD'],
+});
+
+export interface FantasyData {
+  seasonID: number;
+  position: string;
+  name: string;
+  fantasyPoints: number;
+}
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse<Array<FantasyData> | string>
+): Promise<void> => {
+  await use(req, res, cors);
+
+  const { season = null, position = null, name = null} = req.query;
```

Comment:
Is there a reason these are preset as `null`? If season was defaulted as the most recent, can't we set position and name as an empty string or just treat them as `undefined` which requires less ` !== null` stuff below